src/components/common/modal/ModalTopImage.js

```diff
@@ -91,6 +98,11 @@ const getStylesFromProps = ({ theme }) => ({
     height: '20vh',
     width: '100%',
   },
+  spending: {
+    width: normalize(176),
```

Comment:
normalize is for text only
maybe you need getRelativeDesign...

Reply:
Now i use
getDesignRelativeWidth(176),
getDesignRelativeHeight(76),